src/components/Redeem.tsx

```diff
@@ -27,7 +27,8 @@ const Redeem: React.FC<Props> = ({ onCancel }) => {
 
   const logoRef = React.useRef<SVGSVGElement>(null);
   const { balance: optionsBalance } = useOptionsBalance(account);
-  const { balance: umaBalance } = useUMABalance(account);
+  const _umaPayout = Number(optionsBalance) * Number(expirationPayout);
+  const umaPayout = !isNaN(_umaPayout) ? _umaPayout.toFixed(1) : "-";
```

Comment:
is this supposed to fail and become '-' if there is no expiration payout yet?

Reply:
The `expirationPayout` should always be there in practice, I've added it to use "-" as a fallback when the `optionsBalance` is fetching and might be undefined. 
So TLDR yes, but I haven't added that line for it but rather for the `optionsBalance`

Reply:
`optionsBalance` has already been parsed with `ethers.utils.formatEther` and is a string so I would say its safe